handbook/engineering/search/index.md

```diff
@@ -2,9 +2,92 @@
 
 We own the end-to-end Sourcegraph search experience.
 
+## Vision
+
+Sourcegraph search is:
+
+- **Fast:** Performance of showing results is fast and I can get to what I’m looking for quickly.
+- **Scalable:** Sourcegraph scales to any private codebase and the size of the open-source universe. Users can trust that the result set is drawn from the entire set of code they care about, and if the result isn't found, it doesn't exist.
+- **Expressive:** It is possible to construct a search query to find exactly the results I’m looking for. The tools, syntax, filters, are provided to construct the queries I want to write. We strive for a search experience where the answer to “Can I do ‘X’ with Sourcegraph?” will always be yes.
+- **Easy to use:** Users can quickly understand how to find what they are looking for and what options are available for searching. The search syntax is clear and intuitive.
+- **Shareable:** Sourcegraph searches are easy to share, and provide team and organization wide value. This in turn creates network effects that compound the value Sourcegraph provides with scale.
+
 ## Goals
 
-Our current goals are documented in the [search team's current tracking issue](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aopen+is%3Aissue+label%3Atracking+label%3Ateam%2Fsearch).
+_Updated 2020-09-05_
+
+## [WIP] Medium term (3-6 months)
+
+### Scalable
+  - **Scale indexed search to 500k repositories**
```

Comment:
I know this is inconsistent with the short term goals below, but I actually think it is better to bring the concrete goal to the forefront (rather than categorizing) and we should do this for all goals (not just this one). If our goals are clearly written the it should be obvious how these contribute to the vision (therefore it isn't valuable to explicitly have a heading).

If you want to merge this PR as-is, we can discuss this idea in a followup PR.

```suggestion
### Scale indexed search to 500k repositories
```

Reply:
Updating the PR to add this suggestion - I agree that clearly written goals should make it obvious how these contribute to the vision. This can be addressed for the other goals in a follow up.